### PR TITLE
[Merged by Bors] - chore(Algebra/Module/Submodule): add some missing `simps`s

### PR DESCRIPTION
**Pull request by eric-wieser**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by alreadydone — COMMENTED** [review]
maintainer merge

**Review by alreadydone — APPROVED** [review]

File: Mathlib/Algebra/Module/Submodule/LinearMap.lean [hunk 1]
```diff
@@ -226,13 +226,15 @@ instance uniqueOfRight [Subsingleton M₂] : Unique (M →ₛₗ[σ₁₂] M₂)
...
 /-- Evaluation of a `σ₁₂`-linear map at a fixed `a`, as an `AddMonoidHom`. -/
+@[simps]
 def evalAddMonoidHom (a : M) : (M →ₛₗ[σ₁₂] M₂) →+ M₂ where
   toFun f := f a
   map_add' f g := LinearMap.add_apply f g a
   map_zero' := rfl
 #align linear_map.eval_add_monoid_hom LinearMap.evalAddMonoidHom
 
-/-- `LinearMap.toAddMonoidHom` promoted to a `AddMonoidHom` -/
+/-- `LinearMap.toAddMonoidHom` promoted to a `AddMonoidHom`. -/
```
Comment (Vierkantor) [inline Mathlib/Algebra/Module/Submodule/LinearMap.lean:236]:
```suggestion
/-- `LinearMap.toAddMonoidHom` promoted to an `AddMonoidHom`. -/
```

**Review by Vierkantor — APPROVED** [review]
Thanks!

bors d+

**Comment by eric-wieser** [comment]
bors merge

**Pull request closed without merging**